docs/integration/KafkaIntegration.md

```diff
@@ -8,104 +8,164 @@ sidebar_position: 1
 To read more see [Schema registry documentation](https://docs.confluent.io/platform/current/schema-registry/security/index.html#governance)
 
-#### Azure-based implementation
+To make sure if your configuration is correct, you can test it with `kafka-avro-console-consumer`, `kafka-avro-console-producer` available
+in Confluent Schema Registry distribution. After you'll get properly working set of properties, you just need to copy it to Nussknacker's configuration.
 
-For Azure-based implementation, firstly you should provide `schema.registry.url` and `schema.group` properties. First one should be
-`https://<event-hubs-namespace>.servicebus.windows.net`, the second one should be the name of schema groups where will be
+#### Azure-based Schema Registry - Connection and Authentication
+
+For Azure-based implementation, firstly you should provide `schema.registry.url` and `schema.group` properties. First one should be the
+`https://<event-hubs-namespace>.servicebus.windows.net` url, the second one should be the name of schema groups where will be
 located all schemas used by Nussknacker.
 
 Regarding authentication, a couple of options can be used - you can provide credential via:
 `azure.tenant.id`, `azure.client.id` and `azure.client.secret` properties, or you can use one of other methods handled
 by Azure's [DefaultAzureCredential](https://learn.microsoft.com/en-us/java/api/overview/azure/identity-readme?view=azure-java-stable#defaultazurecredential).
 For example via Azure CLI or Azure PowerShell.
 
-### Association between schema with topic
+## Messaging
 
-To properly present information about topics and version and to recognize which schema is assigned to version, Nussknacker follow conventions:
-- For Confluent-based implementation it uses [TopicNameStrategy for subject names](https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#subject-name-strategy).
-  It means that it looks for schemas available at `\<topic-name\>-(key or value)`. For example for topic `transactions`, it looks for schemas at `transactions-key` for key and `transactions-value` subject for value
-- In Azure Schema Registry, subjects concept doesn't exist - schemas are grouped by the same schema name. Because of that, Nussknacker introduces
-  own convention for association between schema and topic: schema name should be in format: CamelCasedTopicNameKey for keys and CamelCasedTopicNameValue for values.
-  For example for `input-events` topic, schema name should be named `InputEventsKey` for key or `InputEventsValue` for value. Be aware that it may require change of schema name
-  not only in Azure portal but also inside schema content - those names should be the same to make serialization works correctly
-  
-### Schema and payload types
+To communicate with running Nussknacker's scenarios via Kafka sources and sinks you can use for example standard kafka-cli commands
```

Comment:
---> You can use standard kafka-cli commands to interact (?? communicate sounds a bit odd to me) with kafka source and sink topics used in Nu scenarios. (I think more precise than sources and sinks)

Reply:
ok, changed